.goreleaser.yml

```diff
@@ -16,6 +16,7 @@ builds:
       - -v
     # We don't use internal `GoReleaser` variables because of consistency with `Makefile` and
     # because we can't change it behaviour.
+    # Note: We duplicate "-X" linker parameters for backward compatibility with go1.12
```

Comment:
I've checked build on `go1.12` and `go1.13` with `GO111MODULE=off/on/auto`. With these flags provided nevermind which mode will be used, it will work.